pkg/apiclient/workflowarchive/workflow-archive.proto

```diff
@@ -15,6 +15,7 @@ message ListArchivedWorkflowsRequest {
 message GetArchivedWorkflowRequest {
   string uid = 1;
   string namespace = 2;
+  string name = 3;
```

Comment:
I would recommend creating a new API and deprecating the existing one for supporting backward capability.

Reply:
Is it going to be a breaking change? Is there a way we can support this feature with backward compatibility?

Reply:
The change would not be considered breaking since introducing fields would not break any existing behavior. We're introducing a new `name` field as part of the existing request to allow users to retrieve workflows by name/namespace, but existing users who are retrieving workflow by UID would not be affected.

Reply:
Thanks, @jessesuen Just had a discussion with @terrytangyuan. I request to test the using current SDK with the new Argo server(signature changes) whether it passes the "" for new arguments or user needs to update the new SDK.

Reply:
I tested this with existing SDK against the new Argo Server and existing behavior preserves. Users can still use only UID to retrieve the archived workflow.